wodles/aws/aws-s3.py

```diff
@@ -704,6 +704,30 @@ def reformat_msg(self, event):
         return event
 
 
+class AWSVPCFlowBucket(AWSLogsBucket):
+    """
+    Represents a bucket with AWS CloudTrail logs
+    """
+
+    def __init__(self, *args):
+        AWSLogsBucket.__init__(self, *args)
+        self.service = 'vpcflowlogs'
+
+    def reformat_msg(self, event):
+        debug('++ Reformat message', 3)
+        AWSBucket.reformat_msg(self, event)
```

Comment:
I'd remove this method and remove the `NotImplementedError` in `AWSLogsBucket` class. That way, the function will automatically call `AWSBucket` implementation.